### PR TITLE
hwdb: ieee1394-unit-function: add Tascam IF-FW/DM mkII

### PR DESCRIPTION
**Pull request by takaswie**
Jörg M. Sigle contriutes configuration ROM image for Tascam IF-FW/DM mkII, which includes the typical content in TCAT Dice-based devices.

This commit adds hwdb entry for the model.

### CODE REVIEW

**Comment by takaswie** [comment]
I realized that thee content includes a quirk. Both node and unit has model_id fields, while they have the different values.

Let me revise this PR as WIP, sorry to puzzle you...

**Comment by takaswie** [comment]
> I realized that thee content includes a quirk. Both node and unit has model_id fields, while they have the different values.

I've checked that the quirk has no effect to the existent implementation for the device attributes, then drop `WIP` from this PR. Please review and proceed it.

Thanks

File: hwdb.d/80-ieee1394-unit-function.hwdb [hunk 1]
```diff
@@ -850,6 +850,13 @@ ieee1394:ven000050C2mo00000070sp000050C2ver00000001
  ID_MODEL_FROM_DATABASE=Duende FireWire
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+ieee1394:node:ven0x00022emo0x800006units0x00022e:0x800006
+ieee1394:ven0000022Emo00800006sp0000022Ever00800006
+  ID_VENDOR_FROM_DATABASE=TASCAM
+  ID_MODEL_FROM_DATABASE=IF-FW/DM mkII
+  IEEE1394_UNIT_FUNCTION_MIDI=1
+  IEEE1394_UNIT_FUNCTION_AUDIO=1
```
Comment (yuwata) [inline hwdb.d/80-ieee1394-unit-function.hwdb]: Please use single space indentation, there seem to two.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/80-ieee1394-unit-function.hwdb (hunk 1 above)
Comment (takaswie) [reply hwdb.d/80-ieee1394-unit-function.hwdb]: > Please use single space indentation, there seem to two.

Oops. I forgot absolutely that the format of hwdb uses one space indentation... Furthermore, some bugs of my tool brings invalid output for the modalias. I correct them and force-pushed the renewed change.

Thanks for your reviewing.

File: hwdb.d/80-ieee1394-unit-function.hwdb [hunk 2]
```diff
@@ -850,6 +850,15 @@ ieee1394:ven000050C2mo00000070sp000050C2ver00000001
  ID_MODEL_FROM_DATABASE=Duende FireWire
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+# The value of model field in unit directory is different from the one in root directory. The value
+# in unit directory is prior to for the modalias.
```
Comment (yuwata) [inline hwdb.d/80-ieee1394-unit-function.hwdb]: I cannot follow the comment. What is 'unit directory'? Is it in another repository?
Please extend the comment, or add a reference about the 'directory'.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/80-ieee1394-unit-function.hwdb (hunk 2 above)
Comment (takaswie) [reply hwdb.d/80-ieee1394-unit-function.hwdb]: Indeed. I wrote it under the context of IEEE 1212, so nowadays few developers get it as is... I'll rewrite it.

Comment (takaswie) [reply hwdb.d/80-ieee1394-unit-function.hwdb]: @yuwata I rewrite the comment and commit message, then force-pushed. Would you please review them?

**Pull request merged**